CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
 - The GDK now uses SpatialOS 14.6.1.
-- Add ability to disable outgoing RPC queue timeouts by setting `QueuedOutgoingRPCWaitTime` to 0.0f.
-- Added `bWorkerFlushAfterOutgoingNetworkOp` (defaulted false) which publishes changes to the GDK worker queue after RPCs and property replication to allow for lower latencies. Can be used in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency at a trade-off with bandwidth.
+- You can now disable outgoing RPC queue timeouts by setting `QueuedOutgoingRPCWaitTime` to 0.0f.
+- We've added `bWorkerFlushAfterOutgoingNetworkOp` (default false) which publishes changes to the GDK worker queue after RPCs and property replication, to allow for lower latencies. You can use this in conjunction with `bRunSpatialWorkerConnectionOnGameThread` to get the lowest available latency at a trade-off with bandwidth.
 
 ## Bug fixes:
-- Fixed a bug that caused queued RPCs to spam logs when an entity is deleted.
-- Take into account OverrideSpatialNetworking command line argument as early as possible (LocalDeploymentManager used to query bSpatialNetworking before the command line was parsed).
-- Servers maintain interest in AlwaysRelevant Actors.
-- GetActorSpatialPosition now returns last spectator sync location while player is spectating.
+- Queued RPCs no longer spam logs when an entity is deleted.
+- We now take the `OverrideSpatialNetworking` command line argument into account as early as possible (previously, `LocalDeploymentManager` queried `bSpatialNetworking` before the command line was parsed).
+- Servers now maintain interest in `AlwaysRelevant` Actors.
+- `GetActorSpatialPosition` now returns the last spectator sync location while the player is spectating.
 - The default cloud launch configuration is now empty.
-- Fixed an crash caused by attempting to read schema from an unloaded class.
-- Unresolved object references in replicated arrays of structs should now be properly handled and eventually resolved.
-- Fix tombstone-related assert that could fire and bring down the editor.
-- Actors placed in the level with bNetLoadOnClient=false that go out of view will now be reloaded if they come back into view.
-- Fix crash in SpatialDebugger caused by dereference of invalid weak pointer.
-- Fixed connection error when using spatial cloud connect external.
-- The command line argument "receptionistHost <URL>" will now not override connections to "127.0.0.1".
-- The receptionist will now be used for appropriate URLs after connecting to a locator URL.
+- We've fixed a crash that happened when the GDK attempted to read schema from an unloaded class.
+- We now properly handle (and eventually resolve) unresolved object references in replicated arrays of structs.
+- We've fixed a tombstone-related assert that could fire and bring down the editor.
+- If an Actor that is placed in the level with `bNetLoadOnClient=false` goes out of view, it is now reloaded if it comes back into view.
```

Comment:
What do we mean by going out of view/coming back into view? Can we be more precise?

Reply:
I thought this might be what it was referring to. Just checked out terminology doc and we do have "worker instance's view" in there, so I think it's OK! (I should have checked earlier - I was thinking maybe we were trying to avoid "view".)